service-discovery/static-list/src/main/java/io/smallrye/stork/servicediscovery/staticlist/StaticListServiceDiscovery.java

```diff
@@ -4,11 +4,11 @@
 import java.util.List;
 
 import io.smallrye.mutiny.Uni;
+import io.smallrye.stork.CachingServiceDiscovery;
 import io.smallrye.stork.DefaultServiceInstance;
-import io.smallrye.stork.ServiceDiscovery;
 import io.smallrye.stork.ServiceInstance;
 
-public final class StaticListServiceDiscovery implements ServiceDiscovery {
+public final class StaticListServiceDiscovery extends CachingServiceDiscovery {
```

Comment:
for this, we can leave it as it was, no need to  cache

Reply:
true! you're right